tests/unit/Lance.spec.js

```diff
@@ -0,0 +1,46 @@
+import Lance from '@/components/Lance'
+import { mount } from '@vue/test-utils'
+
+describe('Lance', () => {
```

Comment:
Vc não fez a verificação do valor se o não for o primeiro lance do leilão. Nesse caso vc poderia separar em 2 *describes* um pra um "leilão que não tem lances" e outro para "um leilão que já tem lances"

Reply:
No caso do lance, ele não se preocupa se já tem lances ou não.  Essa responsabilidade ficou pro leiloeiro. o Lance só recebe como property o valor do lance mínimo. 

acha interessante mudar isso?

Reply:
Entendi, não tem problema então. Só teria que ver pq quando o leiloeiro recebe um novo lance esse valor mínimo teria que ser atualizado em todos os lances. Pensando que tem clientes diferentes conectando no leilão, como vc trata isso?